@vates/nbd-client/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "private": false,
+  "name": "@vates/nbd-client",
+  "homepage": "https://github.com/vatesfr/xen-orchestra/tree/master/@vates/nbd-client",
+  "bugs": "https://github.com/vatesfr/xen-orchestra/issues",
+  "repository": {
+    "directory": "@vates/nbd-client",
+    "type": "git",
+    "url": "https://github.com/vatesfr/xen-orchestra.git"
+  },
+  "author": {
+    "name": "Vates SAS",
+    "url": "https://vates.fr"
+  },
+  "license": "ISC",
+  "version": "0.0.1",
+  "engines": {
+    "node": ">=14.0"
+  },
+  "dependencies": {
+    "@vates/async-each": "^1.0.0",
+    "@xen-orchestra/async-map": "^0.1.2",
+    "promise-toolbox": "^0.21.0",
+    "xen-api": "^1.2.2"
+  },
+  "devDependencies": {
+    "tap": "^16.3.0",
```

Comment:
is it the new way to do it ? I think we shuold better use a specific extension for `tap` test and update global package.json to launch jest test and tap test with the right runner

Reply:
It's a convention that Node packages expose a `test` script to run their own tests.

In XO, we also have the notion of `test-integration` which is used for bigger/slower tests, tests that are usually not self-contained, ie, tests which require installation of specific software on the host machine.

Because it is the case for these tests, I suggest to follow the same principle.

Regarding the fact that some of the current tests are run from the root by Jest, it was done to try to increase the speed of running tests, which, IMHO, is not really necessary anymore with tap.